packaging/common/etc/rundeck/rundeck-config.properties

```diff
@@ -9,3 +9,5 @@ grails.serverURL=http://localhost:4440
 dataSource.dbCreate = update
 dataSource.url = jdbc:h2:file:/var/lib/rundeck/data/rundeckdb;MVCC=true
 rundeck.log4j.config.file = /etc/rundeck/log4j.properties
+
+rundeck.features.repository.enabled=true
```

Comment:
the other feature toggle configurations start with `rundeck.feature.NAME` (singular "feature")